docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,104 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
+* Prevent hanging query execution on failures with `phased` execution policy.
+  ({issue}`10656`)
+* Fix `UnsupportedOperationException` in `max_by` and `min_by` aggregation.
+  ({issue}`10599 `)
+* Fix error message for function calls with lambda arguments. ({issue}`10423 `)
+
+## JDBC driver
+
+* Fix memory leak when using `DatabaseMetaData`. ({issue}`10584 `,
+  {issue}`10632`)
+
+## BigQuery connector
+
+* Fix query failure when reading columns with `numeric` or `bignumeric` type.
+  ({issue}`10564`)
+
+## ClickHouse connector
+
+* Add support for [renaming schemas](/sql/alter-schema). ({issue}`10558`)
+* Add support for setting [column comments](/sql/comment). ({issue}`10641`)
+* Map ClickHouse `ipv4` and `ipv6` types to Trino `ipaddress` type.
+  ({issue}`7098`)
+* Allow mapping ClickHouse `fixedstring` or `string` as Trino `varchar` via the
+  `map_string_as_varchar` session property. ({issue}`10601`)
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Druid connector
+
+* Fix spurious errors when metadata cache is enabled. ({issue}`10544`,
+  {issue}`10512`)
+
+## Hive connector
+
+* Add support for writing Bloom filters in ORC files. ({issue}`3939`)
+* Allow flushing cache for specific partition with the
+  :func:`flush_metadata_cache` function. ({issue}`10385`)
```

Comment:
The link ``` :func:`flush_metadata_cache` ``` is broken.

Reply:
updated to add anchor and change over to the md syntax .. sigh